source/normalize-arguments.ts

```diff
@@ -1,3 +1,4 @@
+import {URL, URLSearchParams} from 'url';
```

Comment:
Please add a TODO comment in the same line to remove this when TypeScript gets fixed

Reply:
I think it would better to open an issue about this as adding a TODO comment to every single import would be noisy.